app/src/main/kotlin/com/wire/android/ui/common/textfield/WireTextField.kt

```diff
@@ -143,7 +202,7 @@ private fun Label(
 }
 
 @Composable
-private fun InnerText(
+ fun InnerText(
```

Comment:
Is there any specific reason why it isn't private?

Reply:
not really :P will revert it